Gemfile

```diff
@@ -46,7 +46,7 @@ group :development do
 end
 
 gem "autoprefixer-rails", "~> 8.4"
-gem "bootstrap-sass", "3.3.5"
+gem "bootstrap-sass", ">= 3.4.1"
```

Comment:
`>=` is dangerous; why `>=` instead of `~>`

Reply:
Why is `>=` considered dangerous? It is what was recommended by Github.

Reply:
I'm guessing because it can pull in `4.0.0` instead of sticking to `3.4.x`.  If you commit Gemfile.lock then danger should be removed as long as no one merges a Gemfile.lock update that goes too far in terms of versions.

Reply:
That makes sense.